nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/ui/ConanBrowseNodeGenerator.java

```diff
@@ -41,29 +57,42 @@ public ConanBrowseNodeGenerator() {
   public List<BrowsePaths> computeAssetPaths(final Asset asset, final Component component) {
     checkNotNull(asset);
 
-    if(component != null) {
+    if (component != null) {
       List<BrowsePaths> strings = computeComponentPaths(asset, component);
       strings.addAll(assetSegment(asset.name()));
       return strings;
-    } else {
+    }
+    else {
       return super.computeAssetPaths(asset, component);
     }
```

Comment:
I guess it will be much better because a component always is null
 return super.computeAssetPaths(asset, null);

Reply:
Fixed 796e51ba6438787c8149e4c5fe30b1b4da43ce54